services/prefect/flows/generated_flows.py

```diff
@@ -133,18 +137,45 @@ def create_cdc_all_states_flow():
     return flow
 
 
+def create_main_flow(flows: List[Flow], project_name):
+    schedule = CronSchedule("0 */3 * * *")
+
+    with Flow("MainFlow", schedule) as main_flow:
+        tasks = []
+        for flow in flows:
+            task = StartFlowRun(
+                flow_name=flow.name, project_name=project_name, wait=True
+            )
+            tasks.append(task)
+
+        parquet_flow = StartFlowRun(
+            flow_name="UpdateParquetFiles", project_name=project_name, wait=True
+        )
+
+        for task in tasks:
+            task.set_downstream(parquet_flow)
```

Comment:
Does this line mean parquet will only run if all scraper flows succeed, or does that adjustment you make to the parquet flow function `trigger=triggers.all_finished` make it so it just waits for all to _finish_ success or not?

Reply:
yup! that's correct - will wait to trigger that until all the upstream tasks are finished (not based on success or failure)

Reply:
Let's give it a shot ;)